packtools/sps/validation/related_articles.py

```diff
@@ -64,26 +65,23 @@ def related_articles_matches_article_type_validation(self, correspondence_list=N
 
         if expected_values_for_related_article_type:
             for related_article in self.related_articles:
-                validated = related_article.get('related-article-type') in expected_values_for_related_article_type
-                yield {
-                        'title': 'Related article type validation',
-                        'xpath': './article[@article-type] .//related-article[@related-article-type]',
-                        'validation_type': 'match',
-                        'response': 'OK' if validated else 'ERROR',
-                        'expected_value': expected_values_for_related_article_type,
-                        'got_value': related_article.get('related-article-type'),
-                        'message': 'Got {}, expected {}'.format(
-                            related_article.get('related-article-type'),
-                            'one of the following items: {}'.format(expected_values_for_related_article_type)
-                        ),
-                        'advice': None if validated else 'The article-type: {} does not match the '
-                                                         'related-article-type: {}, provide '
-                                                         'one of the following items: {}'.format(
-                            self.article_type,
-                            related_article.get('related-article-type'),
-                            expected_values_for_related_article_type
-                        )
-                    }
+                obtained_related_article = related_article.get('related-article-type')
+                is_valid = obtained_related_article in expected_values_for_related_article_type
+                yield format_response(
+                    title='Related article type validation',
+                    parent=None,
+                    parent_id=None,
+                    item='related-article',
+                    sub_item='related-article-type',
+                    validation_type='match',
```

Comment:
@Rossi-Luciano value in list. Se está comparando um valor em uma lista é value in list